cypress/e2e/title-level-request/user-can-move-title-level-request-from-one-item-to-another.cy.js

```diff
@@ -0,0 +1,166 @@
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        instanceData.itemsData = [
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: ITEM_STATUS_NAMES.AVAILABLE },
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: ITEM_STATUS_NAMES.AVAILABLE },
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+        ];
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: testData.instanceTypeId,
+            title: instanceData.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: testData.holdingTypeId,
+              permanentLocationId: testData.defaultLocation.id,
+            },
+          ],
+          items: instanceData.itemsData,
+        }).then((specialInstanceIds) => {
+          instanceData.instanceId = specialInstanceIds.instanceId;
+          instanceData.holdingId = specialInstanceIds.holdingIds[0].id;
+          instanceData.itemIds = specialInstanceIds.holdingIds[0].itemIds;
+        });
+        cy.wrap(instanceData.itemsData).as('itemsToMove');
+      });
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+
+    cy.createTempUser([Permissions.requestsAll.gui], patronGroup.name)
+      .then((userProperties) => {
+        userData = userProperties;
+        UserEdit.addServicePointViaApi(
+          testData.userServicePoint.id,
+          userData.userId,
+          testData.userServicePoint.id,
+        );
+      })
+      .then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+        Requests.createNewRequestViaApi({
+          fulfillmentPreference: FULFILMENT_PREFERENCES.HOLD_SHELF,
+          instanceId: instanceData.instanceId,
+          pickupServicePointId: testData.userServicePoint.id,
+          requestDate: new Date(),
+          requestLevel: REQUEST_LEVELS.TITLE,
+          requestType: REQUEST_TYPES.PAGE,
+          requesterId: userData.userId,
+        }).then((request) => {
+          testData.requestId = request.body.id;
+          testData.currentItem = request.body.item.barcode;
+        });
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    cy.loginAsAdmin({
```

Comment:
Is this action required for changing request status?

Reply:
As I understand we should have permission and I just follow the work follow when change  title level request status in other guy's code

Reply:
Let's leave it like this